package.json

```diff
@@ -3,19 +3,11 @@
   "version": "1.0.0-alpha.1",
   "description": "Chill Dashboard",
   "scripts": {
-    "start": "run-p start:app start:server",
-    "start:app": "webpack-dev-server",
-    "start:server": "nodemon --watch .env --watch src/server --exec babel-node src/server --source-maps",
+    "start": "webpack-dev-server",
     "clean": "rimraf dist",
-    "build": "NODE_ENV=production run-s lint clean build:all",
```

Comment:
Don't remove it. Just keeping it as `NODE_ENV=production run-s lint clean build:app` would be fine.